.github/ISSUE_TEMPLATE/92_tech_open-review.md

```diff
@@ -0,0 +1,31 @@
+---
+name: "[Tech Chair -> Open Practices Chair] Open Materials Review"
+about: Review for Open Materials Chair to fill out
+title: "[REVIEW] Open Materials Review"
+labels: review
+assignees: ''
+---
+
+<!--
+## DO NOT EDIT THIS FILE OUTSIDE OF THE journalovi/jovi-workflows REPOSITORY
+##
+## This file is automatically updated in all repositories within the journalovi
+## Github organization whenever the version in journalovi/jovi-workflows is
+## changed, so any other edits will be overwritten. To update this file, make
+## a commit or pull request at https://github.com/journalovi/jovi-workflows
+-->
+
+### Instructions for Open Practices Chair
+- If there is anything in the list below that you are unsure of, you can reply to this issue (after it has been created) and tag the responsible organizers. Organizers' Github IDs are listed on [People page](https://www.journalovi.org/people.html).
+- If anything in the below list is missing (e.g. required paper sections, supplemental materials, etc), tag the authors on this issue and ask them to add those materials.
+- Once authors have provided any missing material (or if no material is missing) and all requirements are met, close this issue.
+
+### Open Materials Checklist
+- [ ] Check that the paper license is CC-BY
+- [ ] Check that the “Research materials” section in the paper exists.
+- [ ] Check that all links to repositories and materials can be reached.
```

Comment:
The position of this item suggests that the links that should be checked are only those in the "Research materials" section. We should be clear about the scope of where to check (all links that occur in the paper or only links listed in this section).

Also, the item 5 of this checklist "Check that links…" seem to suggest that there are further links to check.